using_images/xpaas_images/eap.adoc

```diff
@@ -9,7 +9,120 @@
+
+The location of `_EAP_HOME_` inside a JBoss EAP xPaaS image is: *_/opt/eap/_*.
+
+== Comparing the JBoss EAP xPaaS Image to the Regular Release of JBoss EAP
+
+=== Functionality Differences for OpenShift JBoss EAP xPaaS Images
+
+There are several major functionality differences in the OpenShift JBoss EAP xPaaS image:
+
+* The JBoss EAP Management Console is not available link:#Managing-OpenShift-JBoss-EAP-xPaaS-Images[to manage OpenShift JBoss EAP xPaaS images].
+* The JBoss EAP Management CLI is only bound locally. This means that you can only access the Management CLI of a container from within the pod.
+* Domain mode is not supported. OpenShift controls the creation and distribution of applications in the containers.
+* There is no HornetQ messaging subsystem. The JBoss A-MQ xPaaS image must be used for scenarios that require messaging.
+* The default root page is disabled. You may want to deploy your own application to the root context (as *_ROOT.war_*). 
+
+[[Managing-OpenShift-JBoss-EAP-xPaaS-Images]]
+=== Managing OpenShift JBoss EAP xPaaS Images
+
+A major difference in managing an OpenShift JBoss EAP xPaaS image is that there is no Management Console exposed for the JBoss EAP installation inside the image. Because images are intended to be immutable, with modifications being written to a non-persistent file system, the Management Console is not exposed.
+
+However, the JBoss EAP Management CLI (*_EAP_HOME/bin/jboss-cli.sh_*) is still accessible from within the container for troubleshooting purposes:
+
+----
+$ oc exec -p <pod> -c <container> - /opt/eap/bin/jboss-cli.sh
+----
+
+[WARNING]
+Any configuration changes made using the JBoss EAP Management CLI on a running container will be lost when the container restarts.
+
+link:#Making-Configuration-Changes-EAP[Making configuration changes to the
+JBoss EAP instance inside the JBoss EAP xPaaS image] is different from the process you may be used to for a regular release of JBoss EAP.
+
+=== Unsupported Configurations
+
+The following is a list of unsupported configurations specific to the JBoss EAP xPaaS image:
+
+* Using MySQL in a scaled environment with XA distributed transactions is not recommended. For applications that support both scaling and XA distributed transactions, PostgreSQL is recommended instead.
+// This is based on https://issues.jboss.org/browse/CLOUD-56
+
+== Installing the JBoss EAP xPaaS Image Streams and Application Templates
+
+To use the Red Hat xPaaS middleware images in your OpenShift project, you must first install the image streams and Source-to-Image (S2I) application templates.
+
+. Clone the Application Template repository:
++
+----
+# git clone https://github.com/jboss-openshift/application-templates
+----
+. Create the image streams and S2I templates:
++
+----
+# oc create -f application-templates/jboss-image-streams.json -n openshift
+# oc create -f application-templates/eap/ *.json -n openshift
```

Comment:
we now document the creation of the image streams in our first steps docs:
https://raw.githubusercontent.com/openshift/openshift-docs/master/admin_guide/install/first_steps.adoc

(you can only see it in raw mode because it's if-def's on Enteprise.  but search for xPaaS

we should link to that instead of re-documenting it here.